packages/compass-global-writes/src/store/reducer.ts

```diff
@@ -0,0 +1,26 @@
+import type { Action, Reducer } from 'redux';
+
+export enum ShardingStatuses {
+  /**
+   * No information yet.
+   */
+  NOT_AVAILABLE = 'NOT_AVAILABLE',
```

Comment:
We had NOT_AVAILABLE and NOT_READY in indexes and now after the refactor ([latest PR in review this moment](https://github.com/mongodb-js/compass/pull/6264)) we removed NOT_AVAILABLE and have just NOT_READY whenever things aren't loaded yet.

I mention it just in case we want to standardise on something.

Reply:
thanks for pointing this out. added in [22ff79f](https://github.com/mongodb-js/compass/pull/6273/commits/22ff79f94d4d6d280d285c9df04e113b95165c61)